Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/ProjectedNodeTrait.php

```diff
@@ -650,6 +650,28 @@ public function iExpectThisNodeToHaveTheFollowingSucceedingSiblings(TableNode $e
         });
     }
 
+    /**
+     * @Then /^I expect this node to tag with "(.*)"$/
```

Comment:
"expect the node to tag with" does not really make sense to me. We should find a common nomenclature.
Also: There were already similar behat steps that didn't require API changes IIRC

Reply:
Agreed, I'm not happy with that either. 
The behat steps that I found only let me check for a single tag, as did the API. But here I need to check whether this node uses the given tags, all of them and only them.

Reply:
"I expect this node is tagged with ..." ?

Reply:
I went for
```
I expect the node with aggregate identifier "<foo>" to be explicitly tagged "<bar>"
```
and
```
I expect the node with aggregate identifier "<foo>" to inherit the tag "<bar>"
```

in the [ProjectedNodeTrait](https://github.com/neos/neos-development-collection/blob/7977bac7dc7fb8e4bf1f7accb5668d5e005fa69b/Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/ProjectedNodeTrait.php#L199-L227) and I think we could just re-use those step implementations!?

Reply:
I want to do a more exact comparison, but I adjusted the naming to be more in line